neptune/alpha/internal/init_impl.py

```diff
@@ -203,13 +198,18 @@ def init(
     if capture_stderr and not _experiment.exists(stderr_path):
         _experiment.define(stderr_path, StringSeries([]))
 
-    entrypoint = sys.argv[0] or None if not is_ipython() else None
-    if source_files is None and not is_ipython() and os.path.isfile(sys.argv[0]):
-        entrypoint = os.path.basename(sys.argv[0]).replace(os.sep, '/')
-        source_files = sys.argv[0]
-
-    if entrypoint is not None:
+    if not is_ipython() and os.path.isfile(sys.argv[0]):
+        if source_files is None:
+            entrypoint = os.path.basename(sys.argv[0])
+            source_files = sys.argv[0]
+        else:
+            common_root = get_common_root(get_absolute_paths(source_files))
+            if common_root is not None:
+                entrypoint = normalize_file_name(os.path.relpath(os.path.abspath(sys.argv[0]), common_root))
+            else:
+                entrypoint = os.path.basename(sys.argv[0])
```

Comment:
Entrypoint may still be in sources even if no common root can be found so should probably use absolute path as we do for all source paths, shouldn't we?